internal/web/server.go

```diff
@@ -237,22 +260,61 @@ func (s *server) packageDetail(w http.ResponseWriter, r *http.Request) {
 		"Status":   status,
 		"Manifest": manifest,
 	})
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "An error occurred rendering the package detail page of %v: \n%v\n", pkgName, err)
-	}
+	checkTmplError(err, fmt.Sprintf("package-detail (%s)", pkgName))
 }
 
 func (s *server) supportPage(w http.ResponseWriter, r *http.Request) {
 	if s.support != nil {
-		err := supportPageTmpl.Execute(w, s.support)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "An error occurred rendering the support page: \n%v\n", err)
+		if s.support.BootstrapMissing {
+			http.Redirect(w, r, "/bootstrap", http.StatusFound)
+			return
 		}
+		err := supportPageTmpl.Execute(w, s.support)
+		checkTmplError(err, "support")
 	} else {
 		http.Redirect(w, r, "/", http.StatusFound)
 	}
 }
 
+func (s *server) bootstrapPage(w http.ResponseWriter, r *http.Request) {
+	if r.Method == "POST" {
+		cfg := client.ConfigFromContext(s.ctx)
+		client := bootstrap2.NewBootstrapClient(
```

Comment:
I think it would be ok to move the IsBootstrapped function from internal to pkg. Then we would not need the two imports and could just use bootstrap.

If you prefer to keep it as is, I think the import name should be changed to something like `internalBootstrap`